packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
+
+Run
+
+```sh
+yarn
+yarn start-test
+```
+
+To start testing the codemods
+
+## To add a codemod
+
+Add your codemods to the `./src/mods` folder with `.mod.ts|tsx` as the file type.
+
+## Test with the actual package:
+
+Run
+
+```sh
+yarn build
+```
+
+To build the upgrade.js file
+
+Run
+
+```sh
+npm pack
+```
+
+from the codemods root to create a tar file for testing. Move the created tar file to the package you want to test and run
+
+```sh
+npx <tarFileName>
+```
+
+## Executing a codemod (Prototype)
+
+Make sure that fluent is installed in your package.
+Run
+
+```
+npx @fluentui\codemods
+```
+
+## Todos
+
+- Write a `flag` utility that will enable devs to note when a part of a file needs to be changed, but cannot be done via codemod.
+- Implement a command that will execute all listed codemods on a single file.
+  - Will need to think of a way to specify the order. Maybe something like tasks in Just.
+  - This could be helpful when you want to run a set of codemods based on a single condition, like the presence of a specific import
```

Comment:
Should these be tracked here? It feels like these should probably be tracked in an issue instead of in the README.

Reply:
I'm going to keep them here for now, since this is a prototype/alpha I think it's fine to have more stuff here. It will be removed before it's published.